docs/core/diagnostics/event-counters.md

```diff
@@ -0,0 +1,353 @@
+---
+title: EventCounters in .NET Core
+description: In this article, you'll learn what EventCounters are, how to implement them, and how to consume them.
+ms.date: 07/22/2020
+---
+
+# EventCounters in .NET Core
+
+**This article applies to: ✔️** .NET Core 3.0 SDK and later versions
+
+EventCounters are .NET Core APIs used for lightweight, cross-platform, and near real-time performance metric collection. EventCounters were added as a cross-platform alternative to the "performance counters" of the .NET Framework on Windows. In this article you'll learn what EventCounters are, how to implement them, and how to consume them.
+
+The .NET Core runtime and a few .NET libraries publish basic diagnostics information using EventCounters starting in .NET Core 3.0. Apart from the EventCounters that are provided by the .NET runtime, you may choose to implement your own EventCounters. EventCounters can be used to track various metrics.
+
+EventCounters live as a part of an <xref:System.Diagnostics.Tracing.EventSource>, and are automatically pushed to listener tools on a regular basis. Like all other events on an <xref:System.Diagnostics.Tracing.EventSource>, they can be consumed both in-proc and out-of-proc via <xref:System.Diagnostics.Tracing.EventListener> and EventPipe/ETW (Event Tracing for Windows).
+
+[![EventCounters in-proc and out-of-proc diagram image](media/eventcounters.jpg)](media/eventcounters.jpg#lightbox)
+
+## Runtime counters
```

Comment:
reading through a second time my suggestion was a little wordy, but hopefully the premise is still good with some better wording.

Reply:
There is no complete listing to be found, and it's an implementation detail of various event sources. It's not reference either, as again - this is an implementation detail. I'm fine with capturing the exhaustive list here, since it is useful and cannot be found elsewhere.